packages/api/functions/cognito/auto-confirm-user.js

```diff
@@ -0,0 +1,7 @@
+exports.handler = async (event) => {
+  event.response = {
+    autoConfirmUser: true,
+    autoVerifyEmail: true,
+  }
+  return event
+}
```

Comment:
@sanathkr for product input. I don't care much for verify email. It adds friction to real users. If someone really wants to create fake accounts, they're going to create fake accounts.